src/admin/components/driver/Drivertable.tsx

```diff
@@ -62,8 +73,12 @@ function Drivertable({ filterValue }: driverTableProp) {
             </tr>
           </thead>
           <tbody>
-            {_.map(drivers.list, (driver: Driver, index: number) => (
-              <DriverTableRow user={driver} index={index} key={driver._id} />
+            {_.map(filteredDrivers, (driver: Driver, index: number) => (
+              <DriverTableRow
+                user={driver}
+                index={index}
```

Comment:
No need to pass `index`, we can use `driver.driverId` as key for DriverTableRow's key.